src/app/components/card/card.component.less

```diff
@@ -0,0 +1,65 @@
+@import (reference) './../../../assets/stylesheets/_base';
+@import (reference) './../../../assets/stylesheets/shared/_layout';
+
+.board-card {
+  width: 100%;
+  float: left;
+  padding: 15px;
+  margin: 0 0 5px 0;
+  background: @color-pf-white;
+  .Border(2px, solid, @color-pf-black-400);
```

Comment:
I'd rather see all of this explicitly stated, rather than a mixin, but I can see why you did it this way. Might be hard to find when the linter runs and errors are located.